pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -445,6 +445,9 @@ private void verifyTableConfigs(TableConfig newTableConfig) {
     SegmentsValidationAndRetentionConfig SegmentConfigToCompare = tableConfigToCompare.getValidationConfig();
 
     String newTimeColumnName = newSegmentConfig.getTimeColumnName();
+    if(newTimeColumnName == null) {
+        throw new PinotHelixResourceManager.InvalidTableConfigException("Must provide valid timeColumnName in tableConfig");
+    }
```

Comment:
Can we revert these 3 lines?

Reply:
Yes. This fix is related to issue https://github.com/apache/incubator-pinot/issues/5558